core/src/commonMain/kotlin/com/ustadmobile/core/view/ContentEntryEdit2View.kt

```diff
@@ -22,6 +22,12 @@ interface ContentEntryEdit2View: UstadEditView<ContentEntryWithLanguage>{
 
     var entryMetaData: ImportedContentEntryMetaData?
 
+    var entryCompressed: Boolean
```

Comment:
'Compressed' is in the past tense. Nothing at this stage has been compressed or altered yet. Perhaps it can be called compressionEnabled

Reply:
If this is going to be a var (which makes sense as it is a view property), then it should be on the view binding, and it should use two way binding.

e.g. 

get() = mBinding?.compressionEnabled ?: false
set(value) {
   mBinding?.compressionEnabled = value
}

Reply:
done